styles.css

```diff
@@ -0,0 +1,500 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --l-gray: gainsboro;
+  --silver: #c0c0c0;
+  --dark-gray: #a9a9a9;
+  --gray: #808080;
+  --blue-gray: #708090;
+  --green-gray: #2f4f4f;
+  --black: #000;
+}
+
+#wrapper {
+  display: grid;
+  max-width: 1440px;
+  grid-template-columns: repeat(5, 1fr);
+  grid-template-rows: minmax(100px, auto);
+  margin: 0 auto;
+  grid-template-areas:
+    "header header header header header"
+    "main main main main main"
+    "section section section section section"
+    "footer footer footer footer footer";
+}
+
+body {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+nav ul {
+  display: flex;
+  justify-content: space-between;
+  padding: 15px 10px;
+  text-align: center;
+  align-items: center;
+}
+
+header {
+  background-color: var(--green-gray);
+  grid-area: header;
+}
+
+a {
+  text-decoration: none;
+  color: white;
+}
+
+.title-section {
+  text-align: center;
+  padding: 40px;
+  background-color: var(--dark-gray);
+}
+
+.title-flex-1 {
+  margin-top: -25px;
+  float: right;
+}
+
+.title-img {
+  width: 30px;
+  background-color: gray;
+  padding: 10px 20px;
+  border-radius: 10px;
+  margin-right: 10px;
+}
+
+footer ul {
+  display: flex;
+  flex-wrap: wrap;
+  width: 80%;
+  margin: 0 auto;
+  justify-content: space-evenly;
+}
+
+footer ul li {
+  margin: 0 15px;
+}
+
+.navlink li {
+  list-style: none;
+  color: white;
+  font-size: large;
+  font-weight: 600;
+}
+
+.navlinka {
+  padding: 5px 10px 7px 10px;
+}
+
+.navlinka:hover {
+  height: 30px;
+  background-color: var(--dark-gray);
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.navlinkmore {
+  background-color: var(--black);
+  padding: 5px;
+  border-radius: 5px;
+}
+
+.logoimage-style {
+  height: 100%;
+  width: 300px;
+  font-size: 30px;
+}
+
+.searchinput-style {
+  padding: 15px 55px;
+  border: 1px solid var(--gray);
+  border-radius: 12px;
+}
+
+.sarchicon-style {
+  position: relative;
+}
+
+.fa-search {
+  position: absolute;
+  left: 3px;
+  color: var(--gray);
+  top: 15px;
+}
+
+main {
+  grid-area: main;
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+}
+
+.flexcontainer {
+  display: flex;
+  align-items: center;
+}
+
+.flexcontainer > * {
+  margin: 0 8px;
+}
+
+.flexcontainer-1 {
+  display: flex;
+}
+
+.class1 {
+  padding: 90px 20px;
+  color: white;
+  display: flex;
```

Comment:
## Using the Same Rule Repetitively[1]
- [ ] Create a class once and the use in each HTML tag where you need it.
- [ ] Delete all your `display:flex;` properties in entire CSS file
- [ ] Add your HTML tags

**Example:**

**CSS file**

```
.flex{
  display:flex;
}
```
**Html file**

```
<div class="flex">
```

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
It's done.